.github/workflows/create-release.yaml

```
@@ -99,6 +103,14 @@ jobs:
           RELEASE_ID=$(./.github/scripts/create-draft-release.sh ${{ github.event.inputs.name }})
           echo "release_id=$RELEASE_ID" >> $GITHUB_OUTPUT
 
+      - name: Render module-config
```

```suggestion
      - name: Render community-module-config
```

Improves readibility of the workflow